foreshadow/metrics/metrics.py

```diff
@@ -1,47 +1,66 @@
 """Metrics used across Foreshadow for smart decision making."""
 
+from foreshadow.core import logging
 
-class Metric(object):  # Metric wrapper
-    """Metric class for metric functions.
 
-    Apply this class by using the metric decorator.
-    """
+class MetricWrapper:
+    """MetricWrapper class for metric functions.
 
-    def __init__(self, fn):
-        """Initialize metric function with fn as attribute.
+    Note:
+        Apply this class by using the metric decorator.
 
-        Args:
-            fn: metric function
-        """
+    Params:
+        fn: Metric function to be wrapped
+        default_return (bool): The default return value of the wrapped
+            function.
+
+    .. automethod:: __call__
+
+    """
+
+    def __init__(self, fn, default_return=None):
         self.fn = fn
+        self.default_return = default_return
 
-    def __call__(self, feature, **kwargs):
+    def __call__(self, feature, invert=False, **kwargs):
```

Comment:
Putting invert here means that whoever calls the metric will say whether or not to invert it. This means the caller needs to have knowledge of the correct direction for what is better - I still think the boolean return might be better.

Reply:
Not sure what you mean.